lbrynet/lbrynet_daemon/ExchangeRateManager.py

```diff
@@ -38,13 +44,12 @@ def __init__(self, market, name, url, params, fee):
         self.rate = None
         self._updater = LoopingCall(self._update_price)
 
+    def rate_updated(self):
```

Comment:
I don't think it needs to be a property, external classes do not need a setter for it, but rate_is_initialized() is a better name